core/settings.py

```
@@ -0,0 +1,652 @@
+                "BACKEND": "django_prometheus.cache.backends.redis.RedisCache",
+                "LOCATION": os.getenv("REDIS_CACHE_TOKEN"),
+                "OPTIONS": {
+                    "CLIENT_CLASS": "django_redis.client.DefaultClient",
+                },
+            },
+            "select2": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": f"{os.getenv("REDIS_CACHE_TOKEN")}/2",
+        "OPTIONS": {
+            "CLIENT_CLASS": "django_redis.client.DefaultClient",
+        }
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)